package.json

```
@@ -12,7 +12,13 @@
 	"scripts": {
 		"initial-setup": "yarn && composer install && yarn run update-configs",
 		"update-configs": "TEXTDOMAIN=wporg composer exec update-configs",
-		"wp-env": "wp-env"
+		"wp-env": "wp-env",
+		"env": "wp-env start",
+		"env:stop": "wp-env stop",
+		"env:setup": "wp-env start --update && yarn run env:reset && yarn run env:import && yarn run env:misc",
+		"env:reset": "wp-env clean all && wp-env run cli \"wp site empty --yes\"",
+		"env:import": "wp-env run cli \"bash env/import.sh\"",
+		"env:misc": "wp-env run cli \"bash env/misc.sh\""
```

```suggestion
		"env:misc": "wp-env run cli \"bash env/misc.sh\""
		"env:cli": "wp-env run cli
```

Without this you have to SSH into the container to run arbitrary wp-cli commands

Adding that upstream to `repo-tools` would be nice too. That goes for some of the other suggestions too. I wonder if there's a more DRY & automation-friendly way than forking `repo-tools/templates/*/` 🤔 

Can `scripts` `extend` the scripts from repo-tools, or something like that? Not worth a lot of time right now, but maybe worth an issue to iterate later?

Maybe `/README.md` can just link to `/vendor/wporg/wporg-repo-tools/readme.md` for the common stuff? Or probably the github.com version, so they can see it before installing. `/env/README.md` could do something similar.

---

The suggested change was missing a comma so I added the same line in 77c5887

---

Not sure about the upstream stuff, but could open an issue in `wporg-repo-tools` to explore the possibilities. It also might make sense to wait until we have multiple project repos depending on it before we add a bunch more stuff to it, so we have a better sense of what's needed.

---

👍🏻 , i'll open an issue there to discuss